python/pynessie/model.py

```diff
@@ -164,7 +163,7 @@ class Reference:
     """Dataclass for Nessie Reference."""
 
     name: str = desert.ib(fields.Str())
-    hash_: Optional[str] = desert.ib(fields.Str(data_key="hash"))
```

Comment:
why lose the optional when we have made it more clear it can be optional?

Reply:
it's back